packages/manager/apps/hycu/src/components/Description/Description.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
To delete and replace by `packages/manager-react-components/src/components/typography/description/description.component.tsx`